index.html

```diff
@@ -46,38 +45,68 @@ <h1 class="display-4"></h1>
     </div>
     </div>
 
-<div class="d-flex justify-content-center">
-<h1>Main Entree's</h1>
+<div class="d-flex container">
+<div class="center-block">
+  <h2>Main Entree's</h2>
+</div>
 </div>
  
 <!--Speakers/Dishes need rows-->
-<div class="d-flex container justify-content-center p-3">
+<div class="d-flex container justify-content-center ">
   <div class="row">
-    <div class="col-sm">
-      <img  src="img/roberto-carlos-roman-TS_g_856-CA-unsplash.jpg" class="img-thumbnail" alt="Cinque Terre">
-      <h1>De La Flor</h1>
+    <div class="col-sm-3">
+      <img  src="img/roberto-carlos-roman-TS_g_856-CA-unsplash.jpg" class="img-thumbnail" alt="Food Plate of Locoro rice and Beans">
+      <h2>De La Flor</h2>
   <p> Flor de Locoro, Arroz Blanco, y Frijoles Fritos Pintos.</p>
     </div>
-    <div class="col-sm">
-      <img src="img/christine-siracusa-vzX2rgUbQXM-unsplash.jpg" class="img-thumbnail" alt="Cinque Terre">
-      <h1>Tacos</h1>
+    <div class="col-sm-3">
+      <img src="img/christine-siracusa-vzX2rgUbQXM-unsplash.jpg" class="img-thumbnail" alt="Food PLate of Beef Tacos ">
+      <h2>Tacos</h2>
   <p></p>
   <p>Carne de Asada tacos con opciones de chile verde e rojo con verdura.</p>
     </div>
-    <div class="col-sm"> 
-      <img src="img/ismael-trevino-Y0zbn9lPCEU-unsplash.jpg" class="img-thumbnail" alt="Cinque Terre">
-      <h1>Nachos Jaliscense's</h1>
+    <div class="col-sm-3"> 
+      <img src="img/ismael-trevino-Y0zbn9lPCEU-unsplash.jpg" class="img-thumbnail" alt="Nachos of Beef">
+      <h2>Nachos Jaliscense's</h2>
   <p>Red chile carne asada nachos with white maiz tortilla chips.</p>
     </div>
-    <div class="col-sm">
-      <img src="img/jennifer-griffin-1yDF6qRULCY-unsplash.jpg" class="img-thumbnail" alt="Cinque Terre">
-      <h1>Sopes</h1>
+    <div class="col-sm-3">
+      <img src="img/jennifer-griffin-1yDF6qRULCY-unsplash.jpg" class="img-thumbnail" alt="Food Plates of Sopes and Rice">
+      <h2>Sopes</h2>
   <p>Sopes chicos con arroz mexicano.</p>
   </div>
   </div>
  </div>
 
+ <!--Get Your Ticket Today-->
+ <div class="d-flex container btn-vertical">
+  <h1>Get Your Grand Opening Reservation Today!</h1>
+  <p>Each Reservation includes complimentary grand opening surprise desert.</p>
```

Comment:
The reservation part doesn't look balanced on the same level as the other objects.
![Screenshot (11)](https://user-images.githubusercontent.com/25779736/98649096-08c9c800-22ec-11eb-8216-485319f7ae3f.png)